packages/frontend-core/src/utils/lucene.js

```diff
@@ -211,6 +212,11 @@ export const runLuceneQuery = (docs, query) => {
     return docValue == null || docValue === ""
   })
 
+  // Process an includes match (fails if the value is not included)
+  const oneOf = match("oneOf", (docValue, testValue) => {
+    return !testValue?.includes(docValue)
```

Comment:
Looks great @melohagan! Could we copy the logic here to support splitting strings into arrays if the value is a string? The same way I see that it gets handled server side. Would be nice to keep support consistent, otherwise comma separated strings will work with internal/SQL but not with custom queries.